resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content

```diff
@@ -1 +1 @@
-.featured{font-size:.8rem;line-height:1.5;color:#555;text-align:center}.featured h3{margin-top:10px;margin-bottom:10px;font-weight:400;color:#333;text-align:center}.dark .featured,.dark .featured h3{color:#fff}.featured-icon{display:block;width:100%;height:40%;font-size:3rem;text-align:center}.featured-icon img{margin:0 auto}.featured .col-sm-4 p{text-align:justify}
```

Comment:
the resources files needs to be skipped

Reply:
just need to be skipped ? or we can remove from the repo?

> the resources files needs to be skipped

just need to be skipped ? or better we can remove from the repo?

Reply:
as discussed deleted all under _gen/css & _gen/js folders.